addons/addon-base-raas/packages/base-raas-services/lib/environment/service-catalog/environment-sc-service.js

```diff
@@ -307,10 +309,12 @@ class EnvironmentScService extends Service {
     let instanceType;
     let instanceIdentifier;
     const outputsObject = cfnOutputsArrayToObject(outputs);
-    // TODO: Remove MetaConnection1Type check to include RStudio after CNAME patch
     if ('Ec2WorkspaceInstanceId' in outputsObject && _.get(outputsObject, 'MetaConnection1Type') !== 'RStudio') {
       instanceType = 'ec2';
       instanceIdentifier = outputsObject.Ec2WorkspaceInstanceId;
+    } else if ('Ec2WorkspaceInstanceId' in outputsObject && _.get(outputsObject, 'MetaConnection1Type') === 'RStudio') {
```

Comment:
Minor - this could be combined to the previous if block? Since they both have 'Ec2WorkspaceInstanceId' in outputsObject.

Reply:
Yeah, but I purposely kept it separate for better code readability.